content-src/experiments/activity-stream.yaml

```diff
@@ -18,6 +18,9 @@ xpi_url: 'https://moz-activity-streams.s3.amazonaws.com/dist/activity-streams-la
 addon_id: '@activity-streams'
 gradient_start: '#0996F8'
 gradient_stop: '#0675D3'
+min_release:
```

Comment:
We should probably build the front-end in such a way that we don't need to include empty keys at all.